### PR TITLE
[READY] blog Koos about Privacywet

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
A short blog about the privacywet from Koos. Please review @tarzan @dkhgh @fatboypunk

### CODE REVIEW

File: source/blog/2016-02-22-nieuwe-privacywet.html.markdown [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Wetswijziging meldplicht datalekken
+date: 2016-02-22 7:30 CET
+tags:
+author: Koos 
+lang: nl
+featured: true
```
Comment (fatboypunk) [inline source/blog/2016-02-22-nieuwe-privacywet.html.markdown]: Do we want this to be a featured blog?

**Comment by fatboypunk** [comment]
Might be nice to add a link with more info about this law

File: source/blog/2016-02-22-nieuwe-privacywet.html.markdown (hunk 1 above)
Comment (RonaldDijkstra) [inline source/blog/2016-02-22-nieuwe-privacywet.html.markdown]: No, will change it tomorrow

**Comment by RonaldDijkstra** [comment]
Good idea!

**Comment by RonaldDijkstra** [comment]
- [x] Removed featured
- [x] Added a link to more information about the changed law

**Comment by sn3p** [comment]
:+1:

**Pull request merged**